src/libs/antares/study/study.h

```diff
@@ -664,11 +664,6 @@ class Study: public Yuni::NonCopyable<Study>, public IObject, public LayerData
     void* cacheTSGenerator[timeSeriesCount];
     //@}
 
-    /*!
-    ** \brief
-    */
-    bool gotFatalError = false;
```

Comment:
100% agreed it had nothing to do here: it should have been the result of a loading function, not a class member